### DISCUSSION TITLE
I need assistance to recover an account

### DISCUSSION THREAD

**Discussion by aasilghoury** [community/community#138726] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Problem Overview

I have two GitHub accounts. The one I've been using for two years has the username @lalaghoury. I want to change this username to @aasilghoury (my real name) now that I am a working professional and want to use my actual name for contributions.

### Main Issue

I’m facing the following problem:
- I can’t remember the email address I used to create the @aasilghoury account, but I do have the password saved in my password manager.
- When I try to log in, GitHub says "invalid credentials."
- When I visit `https://www.github.com/aasilghoury`, it shows a 404 page, which indicates the account doesn’t exist.
- I’m fairly sure I still have access to the email address used for the @aasilghoury account.

Usually, a person's email address is displayed somewhere on their profile page. However, since I can't access the @aasilghoury profile page (due to the 404 error), I can't find the email address on my own. That's why I need assistance from the GitHub team.

### What I Need

If I can find out which email address is associated with the @aasilghoury account, I could log in, delete the account, and then use that username for my @lalaghoury account. Could the GitHub team help me with this?

**Comment by davevad93** [comment, marked as answer]
Hi @lalaghoury ,

For account related issues, it's recommended to contact support, open a ticket [here](https://support.github.com/contact/account), select “`Sign in, verification, or other account questions`” and ask them if they can help you with your problem.

**Reply by aasilghoury** [reply]
Thanks brither I opened a ticket for further assistance

**Reply by ebndev** [reply]
Hi @lalaghoury,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!

**Comment by jovanovic-djo** [comment]
@davevad93 Provided good answer.

Additionally, I would suggest you to read through some of the other users experience on reddit or other forums. It might take up to few weeks for them to resolve certain issues.
